modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/MustacheScriptEngine.java

```diff
@@ -47,6 +54,19 @@ public final class MustacheScriptEngine implements ScriptEngine {
 
     public static final String NAME = "mustache";
 
+    public static final Setting<ByteSizeValue> MUSTACHE_RESULT_SIZE_LIMIT = new Setting<>(
+        "mustache.result.size.limit",
```

Comment:
nit: we normally use dots for organization/grouping of settings, not word breaks. So I think this should be something like `mustache.max_output_size_in_bytes`